pykusto/functions.py

```diff
@@ -567,11 +568,11 @@ def pack_array(*elements: ExpressionType) -> 'ArrayExpression':
         return ArrayExpression(KQL(f'pack_array({", ".join(to_kql(e) for e in elements)})'))
 
     @staticmethod
-    def pack_dictionary() -> MappingExpression:
+    def pack_dictionary(**kwargs: ExpressionType) -> MappingExpression:
         """
         https://docs.microsoft.com/en-us/azure/data-explorer/kusto/query/packdictionaryfunction
         """
-        raise NotImplemented  # pragma: no cover
+        return MappingExpression(KQL(f'pack_dictionary({", ".join(f"{to_kql(k)}, {to_kql(v)}" for k, v in kwargs.items())})'))
```

Comment:
if it is an alias for pack just call pack or the other way around

Reply:
I think it's best to keep the behavior of Kusto, if they decide to change it one day, it might behave weird